packages/api/src/check/CheckStatus.ts

```diff
@@ -0,0 +1,4 @@
+export enum CheckStatus {
+  Passed = 'passed',
+  CompileError = 'compileError',
```

Comment:
It doesn't throw an error on `,` ending object?

Reply:
You mean it should give a lint error? Guess not for enums.

Reply:
IMO it should. I don't know why it doesn't :/ Have you changed lint settings? (or maybe during update something was broken ._.

Reply:
Should I make PR for it? I have pretty up to date config with mostly the same options like here :)

Reply:
Sure, a serape PR is fine.

Reply:
I think trailing commas are quite convenient.